nnpdf_data/nnpdf_data/new_commondata/NNPDF_INTEG_3GEV/metadata.yaml

```
@@ -92,3 +92,123 @@ implemented_observables:
     - - INTEGXT8
   data_uncertainties: []
   ported_from: INTEGXT8
+- observable_name: XV
+  observable:
+    description: Deep Inelastic Scattering
+    label: 'integrability dataset: $xV$ PDF, Q=Q_0'
+    units: ''
+  process_type: DIS_XV
+  tables: []
+  npoints: []
+  ndata: 1
+  plotting:
+    kinematics_override: dis_sqrt_scale
+    theory_reference: Bertone:2013vaa
+    dataset_label: 'integrability dataset: $xV$ PDF, Q=Q_0'
+    plot_x: k1
+  kinematic_coverage:
+  - k1
+  - k2
+  - k3
```

```suggestion
  - x
  - Q
```

You should be able to get away with defining only two.